docs/openapi/components/schemas/common/Organization.yml

```diff
@@ -115,20 +101,72 @@ properties:
     $linkedData:
       term: iataCarrierCode
       '@id': https://onerecord.iata.org/cargo/Company#airlineCode
+  organizationType:
+    title: Organization Type
+    description: Type of organization (e.g. cold storage, meat processing).
+    type: string
+    $linkedData:
+      term: organizationType
+      '@id': https://w3id.org/traceability#organizationType
+  place:
+    title: Place
+    description: Information on organization place.
+    $ref: ./Place.yml
+    $linkedData:
+      term: place
+      '@id': https://schema.org/Place
+  gap:
```

Comment:
What's GAP? Please spell out acronyms.

Reply:
[Good Agricultural Practices (GAP)](https://www.ams.usda.gov/services/auditing/gap-ghp) but yes this should be in the description at a minimum

Reply:
seems like a subtype specific to ag.

Reply:
@rhofvendahl  GAP will only apply to ag organizations - should not be a top level member of organization - GAP should be an ag inspection in reference to the organization

Reply:
@rhofvendahl you may want an AgOrganization that includes org as a top level member, then adds gap and other certs as required